### DISCUSSION TITLE
Pregunta de enlaces interno

### DISCUSSION THREAD

**Discussion by Asalz20** [reactjs/es.reactjs.org#568] [Q&A] [answered]
En la página de createRoot (y muchas otras) hay enlaces internos que apuntan a otras partes de la página. Cómo puedo renderizar las enlaces usando Markdown? Y deberían traducirse también?

**Comment by carburo** [comment, marked as answer]
No estoy seguro a qué te refieres en la primera pregunta. ¿Podrías abundar un poco más en qué quieres hacer específicamente? Los links se mostrarán correctamente al correr el sitio localmente o con la previsualización del despliegue que se crea al abrir un PR. En un previsualizador de markdown en un editor de texto puede que no funcione, depende de la compatibilidad que tenga con esta [sintaxis extendida](https://www.markdownguide.org/extended-syntax/#heading-ids).

Sobre la segunda pregunta: No, no deben traducirse. En nuestra guía de estilo (que recomendamos leer en su totalidad antes de comenzar a traducir) hay [una sección](https://github.com/reactjs/es.reactjs.org/blob/main/TRANSLATION.md#identificadores-de-los-t%C3%ADtulos) con ejemplos dedicada a este tema.

**Comment by Byron2016** [comment]
Pongo en consideración, que la respuestá debería incluir la solución definitiva, es correcto poner el link al artículo, pero además colocar el código de esta forma ahorra a la persona el tener que visitar el link (lo puede hacer si lo desea)


| ACCIÓN                                                | MARKDOWN                                           | HTML  TRANSFORMADO                                      | 
|-------------------------------------------|--------------------------------------------|-------------------------------------------------------|
| Marcar Un identificador en página x    | `### My Great Heading {#custom-id}`   | `<h3 id="custom-id">My Great Heading</h3>`  | 
| Invocarlo en algún lugar de página x   | `[Heading IDs](#heading-ids)`                | `<a href="#heading-ids">Heading IDs</a>`        |
